src/code/code_80043480.c

```diff
@@ -3,73 +3,73 @@
 void DynaPolyActor_Init(DynaPolyActor* dynaActor, s32 flags) {
     dynaActor->bgId = -1;
     dynaActor->unk_15C = flags;
-    dynaActor->unk_160 = 0;
+    dynaActor->interactFlags = 0;
     dynaActor->unk_150 = 0.0f;
     dynaActor->unk_154 = 0.0f;
 }
 
-void func_800434A0(DynaPolyActor* dynaActor) {
-    dynaActor->unk_160 = 0;
+void DynaPolyActor_ResetInteractFlags(DynaPolyActor* dynaActor) {
+    dynaActor->interactFlags = 0;
 }
```

Comment:
clear instead of reset maybe? i think thats the more common terminology we use anyway

Reply:
"clear" is the proper word for "unset" (a single bit) but okay...

Reply:
I think this should definitely be UnsetAll, Reset or an equivalent, to make it clear.

But we can leave that for a separate PR if you prefer